src/AWSSDK.MobileAnalytics.MobileAnalyticsManager.csproj

```diff
@@ -12,8 +12,7 @@
     <PackageTags>&gt;AWS;Amazon;cloud;CognitoSync;CognitoSyncManager;aws-sdk-v3</PackageTags>
     <PackageProjectUrl>https://github.com/aws/aws-mobile-analytics-manager-net</PackageProjectUrl>
     <PackageLicenseFile>LICENSE</PackageLicenseFile>
-    <!-- Replace with PackageIcon when Visual Studio supports it. -->
-    <PackageIconUrl>https://sdk-for-net.amazonwebservices.com/images/AWSLogo128x128.png</PackageIconUrl>
+    <PackageIcon>icon.png</PackageIcon>
```

Comment:
is the path relative to the csproj?  should the path be `..\icon.png`?  Looks like the file is in the root of the repository.

Reply:
Yup, below in the csproj file it is included given it's path relative to csproj
```
<ItemGroup>
    <None Include="../LICENSE" Pack="true" PackagePath="" />
    <None Include="../icon.png" Pack="true" PackagePath="" />
</ItemGroup>
```

Reply:
Yup, missed that, thanks!